chromadb/test/configurations/test_configurations.py

```diff
@@ -76,3 +77,8 @@ def test_validation() -> None:
     ]
     with pytest.raises(ValueError):
         TestConfiguration(parameters=invalid_parameter_names)
+
+
+def test_hnsw_validation() -> None:
```

Comment:
I would add a generic version of this test which tests configuration-level validators